assets/script.js

```diff
@@ -171,3 +207,47 @@ span.onclick = function() {
     modal.css('display','none');
   }
 } */
+
+// When the user clicks the favourite button, save the card id
+// to local storage and fill in the heart icon, or remove if it's
+// already there
+
+$(document).on("click",".favourite-btns",saveToLocalStorage);
+
+// Save to local storage 
+function saveToLocalStorage (event){
+  event.preventDefault();
+  // modal.css('display','block');
+  let itemID = $(this).attr("id");
+
+  // parse existing storage key or string representation of empty array (uses || operator, means
+  // to take "lis_items" or if that is false take empty array '[]')
+  var existingEntries = JSON.parse(localStorage.getItem("locations") || '[]');
+
+  console.log(existingEntries);
+  console.log(itemID);
+
+
+  // Add item if it's not already in the array, then store array again
+  if (!existingEntries.includes(itemID)) {
+    existingEntries.push(itemID);
+    localStorage.setItem("locations", JSON.stringify(existingEntries));
```

Comment:
This line could also be in its own function and called here and line 241

Reply:
Ok yes good idea, I've done that

Reply:
Great - thanks!